packages/backend/src/routes/courses.ts

```diff
@@ -39,7 +39,35 @@ coursesRouter.get("/:id", async (ctx, next) => {
             );
         });
 
-        ctx.body = rows;
+        const assignments = await new Promise((resolve, reject) => {
```

Comment:
@fravl hmm.. I don't think we should be returning assignments and lectures from this route since its for courses. The client should fetch these entities separately by their respective rotues:

First:
- /courses/:id for course
Then (when course_id is available):
- /assignments/:id for assignments
- /lectures/:id for lectures

The assignments and lectures can be fetched in parallel.